models/sipnet/R/met2model.SIPNET.R

```diff
@@ -44,7 +75,7 @@ met2model.SIPNET <- function(in.path, in.prefix, outfolder, start_date, end_date
                         dbfile.name = out.file,
                         stringsAsFactors = FALSE)
   PEcAn.logger::logger.info("internal results")
-  print(results)
+  print(results) # Not a debugging statement
```

Comment:
Throughout the code, there were a number of `print` statements.  I removed all print statements I introduced for debugging; informational messages were all done through PEcAn.logger.  I (incorrectly) assumed that those were there for a reason.  I can certainly change them to PEcAn.logger statements.

Reply:
They're probably `print`s because

```
> PEcAn.logger::logger.info(iris[1:3,])
2018-07-17 22:27:41 INFO   [console] : 
   c(5.1, 4.9, 4.7) c(3.5, 3, 3.2) c(1.4, 1.4, 1.3) c(0.2, 0.2, 0.2) c(1, 
   1, 1) 
> print(iris[1:3,])
  Sepal.Length Sepal.Width Petal.Length Petal.Width Species
1          5.1         3.5          1.4         0.2  setosa
2          4.9         3.0          1.4         0.2  setosa
3          4.7         3.2          1.3         0.2  setosa

```

Reply:
Actually, I added that print statement in the style of others.  Changed.